ts/components/screens/ListItemComponent.tsx

```diff
@@ -73,65 +77,70 @@ const styles = StyleSheet.create({
 export default class ListItemComponent extends React.Component<Props> {
   public render() {
     return (
-      <ListItem
-        style={[styles.listItem, styles.flexRow, this.props.style]}
+      <TouchableWithoutOpacity
+        style={styles.highlight}
         onPress={this.props.onPress}
         onLongPress={this.props.onLongPress}
```

Comment:
**_TouchableWithoutOpacity_** doesn't have _**onLongPress**_ so i.e. on Service Home long press on items doesn't work anymore.
Please make all tests needed to check if all features are preserved